Sources/SimultaneouslyScrollView/DefaultSimultaneouslyScrollViewHandler.swift

```diff
@@ -2,8 +2,18 @@
 import Combine
 import UIKit
 
+internal class ScrollViewDecorator {
+    weak var scrollView: UIScrollView?
+    var direction: ScrollDirection?
+
+    init(scrollView: UIScrollView, direction: ScrollDirection?) {
+        self.scrollView = scrollView
+        self.direction = direction
+    }
+}
+
 internal class DefaultSimultaneouslyScrollViewHandler: NSObject, SimultaneouslyScrollViewHandler {
-    private var scrollViewsStore: WeakObjectStore<UIScrollView> = WeakObjectStore()
+    private var scrollViewsStore: [ScrollViewDecorator] = []
```

Comment:
This change makes the `WeakObjectStore` redundant and therefore it can be removed.

Reply:
[9b99381](https://github.com/stonko1994/SimultaneouslyScrollView/pull/14/commits/9b99381a4b32f6e4398d75b586ca656a0f638a20)